tensorflow_addons/rnn/cell.py

```diff
@@ -366,3 +366,471 @@ def _create_norm_layer(self, name):
             gamma_initializer=self.norm_gamma_initializer,
             epsilon=self.norm_epsilon,
             name=name)
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+# class LayernormSimpleRNNCell(SimpleRNNCell, LayerNormalization):
+class LayernormSimpleRNNCell(keras.layers.SimpleRNNCell):
+    """Cell class for LayernormSimpleRNN.
+
+    Motivation:
+    - Drop-In Replacement for keras.layers.SimpleRNNCell
+    - demonstrate how to add keras.layers.LayerNormalization
+       to all RNNs by introducing the `use_layernorm` argument
```

Comment:
Maybe I didn't express it clearly in the previous review, but I think it doesn't make sense to have this switch at all. The LayernormSimpleRNNCell should always use layernorm.

Reply:
done